releasenotes/notes/deprecate-dbapi-config-0fc5b1acfc4aded2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    Deprecate `ddtrace.config.dbapi2` as default for `TracedCursor` and `TracedConnection` as well as `DD_DBAPI2_TRACE_FETCH_METHODS`. Use `IntegrationConfig` and `DD_<INTEGRATION>_TRACE_FETCH_METHODS` specific to each dbapi-compliant library.
```

Comment:
should we provide an example here, like:

```python
# Before
config.dbapi2.trace_fetch_methods = True

# After
config.psycopg2.trace_fetch_methods = True
```

?

Reply:
👍 sounds good